Orange/widgets/regression/owsgdregression.py

```diff
@@ -180,6 +185,16 @@ def _on_func_changed(self):
         for spin, enabled in zip(self._func_params, mask):
             spin.setEnabled(enabled)
 
+    def send_report(self):
+        items = OrderedDict()
+        items['Loss function'] = self.loss_function_options[
+            self.loss_function].capitalize()
+        items['Penalty'] = self.penalty_options[self.penalty_type].capitalize()
```

Comment:
`capitalize()` here will lowercase "L1" and "L2" in the string. All presentational strings are properly capitalized already.